src/index.ts

```diff
@@ -1,4 +1,13 @@
+import log from "npmlog";
+
 import server from "./server";
-import { getPort } from "./config";
+import { getPort, getProxyHostPort } from "./config";
+let hostPort;
+try {
+  hostPort = getProxyHostPort();
+} catch (e) {
+  log.error("setup", e.message);
+  process.exit(1);
+}
 
-server(getPort());
+server(getPort(), hostPort);
```

Comment:
This has to be converted to one `MockServerOptions`-Object in the future.

Reply:
Depends on how much configuration is going to be added and if we want to allow SDK-like usage at all IMHO.

Reply:
sure, but I think this will have a lot of config stuff… do you think it will be limited to a few args? 🤔